nva-datamodel-java/src/main/java/no/unit/nva/model/ImportCandidate.java

```diff
@@ -0,0 +1,429 @@
+package no.unit.nva.model;
+
+import static java.util.Objects.hash;
+import static java.util.Objects.nonNull;
+import static nva.commons.core.attempt.Try.attempt;
+import static nva.commons.core.ioutils.IoUtils.stringFromResources;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+import com.github.bibsysdev.ResourcesBuildConfig;
+import java.net.URI;
+import java.nio.file.Path;
+import java.time.Instant;
+import java.util.Collections;
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import no.unit.nva.WithAssociatedArtifact;
+import no.unit.nva.WithIdentifier;
+import no.unit.nva.WithMetadata;
+import no.unit.nva.commons.json.JsonUtils;
+import no.unit.nva.identifiers.SortableIdentifier;
+import no.unit.nva.model.associatedartifacts.AssociatedArtifact;
+import no.unit.nva.model.associatedartifacts.AssociatedArtifactList;
+import no.unit.nva.model.funding.Funding;
+import no.unit.nva.model.funding.FundingList;
+import nva.commons.core.JacocoGenerated;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
+@SuppressWarnings({"PMD.ExcessivePublicCount", "PMD.TooManyFields", "PMD.GodClass"})
+public class ImportCandidate
```

Comment:
Eh, tror du har misforstått. Det trenger ikke å være her i det hele tatt.

ImportCandidate extends publication i publication-api. 
Den trengs ikke å eksponeres for omverden direkte.